### DISCUSSION TITLE
Moving SportsTracker to a new computer

### DISCUSSION THREAD

**Discussion by The-BDD-Coach** [ssaring/sportstracker#265] [Q&A] [answered]
I have installed SportsTracker on a new computer, but the directory structure is very different (Linux versus MacOS). I have copied over the data files, but when I launch SportsTracker it says "The exercise files attached to the following exercises can't be found', and then shows a long list of exercise files.

How can I change the directory where SportsTracker expects to find the exercise files? I am running SportsTracker 8.0.1 on MacOS 14.2.1.

**Comment by ssaring** [comment]
Since SportsTracker 8.0.0 all data is stored in the SQLite database file "sportstracker.sqlite". So the easiest way is to open this database with a SQLite browser / editor of your choice and update all referenced HRM files in the table EXERCISE with an appropriate SQL statement.

The DB schema is defined here: https://github.com/ssaring/sportstracker/blob/master/sportstracker/src/main/resources/sql/st-schema.sql

**Reply by The-BDD-Coach** [reply, marked as answer]
Thank you!

The correct steps are:
1) Install your favorite SQLite DB Browser (I used DB Browser for SQLite, from sqlitebrowser.org.)
2) Open the EXERCISE table; you should see that the HRM_FILE field contains '/home/your user name/...', because we moved the data from a Linux system. However, on MacOS this should be '/Users/your user name/...'.
3) Execute this SQL:
update EXERCISE SET
   HRM_FILE = replace(HRM_FILE, '/home/<your user name>/', '/Users/your user name/')
 WHERE
    HRM_FILE like '/home/your user name/%';

This will change all instances of '/home/your user name/' to '/Users/your user name/'; that should fix the problem.

**Reply by ssaring** [reply]
Great that you was able to solve your problem. And thanks for sharing, this might be helpful for others as well.
